extensions/cloud/portability-cloud-local/src/main/java/org/dataportabilityproject/cloud/local/LocalJobStore.java

```diff
@@ -26,12 +27,15 @@
 import java.util.Map.Entry;
 import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
+import org.slf4j.Logger;
```

Comment:
move this to a separate PR too

also, I thought the logger should come from this extension (the cloud-local extension)'s context (ExtensionContext.getLogger) but it seems we're not doing that elsewhere. MIght want to check with @jimmarino on that but I'm also fine with leaving this alone as it's consistent with how it's currently done elsewhere